src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/integTests/tests/jvm/old_gen_policy/dedicated_master/LevelThreeDedicatedMasterITest.java

```diff
@@ -0,0 +1,178 @@
+            tuple = {
+                @ATuple(dimensionValues = {ThreadPoolType.Constants.WRITE_NAME},
+                    sum = QueueActionConfig.DEFAULT_WRITE_QUEUE_UPPER_BOUND - 200,
+                    avg = QueueActionConfig.DEFAULT_WRITE_QUEUE_UPPER_BOUND - 200,
+                    min = QueueActionConfig.DEFAULT_WRITE_QUEUE_UPPER_BOUND - 200,
+                    max = QueueActionConfig.DEFAULT_WRITE_QUEUE_UPPER_BOUND - 200),
+                @ATuple(dimensionValues = {ThreadPoolType.Constants.SEARCH_NAME},
+                    sum = QueueActionConfig.DEFAULT_SEARCH_QUEUE_UPPER_BOUND - 200,
+                    avg = QueueActionConfig.DEFAULT_SEARCH_QUEUE_UPPER_BOUND - 200,
+                    min = QueueActionConfig.DEFAULT_SEARCH_QUEUE_UPPER_BOUND - 200,
+                    max = QueueActionConfig.DEFAULT_SEARCH_QUEUE_UPPER_BOUND - 200)
+            }
+        )
+    }
+)
+public class LevelThreeDedicatedMasterITest {
+  public static final long HEAP_MAX_SIZE_IN_BYTE = 10 * GB_TO_BYTES;
+  public static final double FIELDDATA_CACHE_SIZE_IN_PERCENT = 0.3;
+  public static final double SHARD_REQUEST_CACHE_SIZE_IN_PERCENT = 0.04;
+
+  @Test
+  @AExpect(
+      what = AExpect.Type.REST_API,
+      on = HostTag.ELECTED_MASTER,
+      validator = LevelThreeValidator.class,
+      forRca = PersistedAction.class,
+      timeoutSeconds = 1000)
+  @AErrorPatternIgnored(
+      pattern = "CacheUtil:getCacheMaxSize()",
+      reason = "Cache related configs are expected to be missing in this integ test")
+  @AErrorPatternIgnored(
+      pattern = "AggregateMetric:gather()",
+      reason = "Cache metrics are expected to be missing in this integ test")
+  @AErrorPatternIgnored(
+      pattern = "SubscribeResponseHandler:onError()",
+      reason = "A unit test expressly calls SubscribeResponseHandler#onError, which writes an error log")
+  @AErrorPatternIgnored(
+      pattern = "SQLParsingUtil:readDataFromSqlResult()",
+      reason = "Old gen metrics is expected to be missing in this integ test.")
+  @AErrorPatternIgnored(
+      pattern = "HighHeapUsageOldGenRca:operate()",
+      reason = "Old gen rca is expected to be missing in this integ test.")
+  @AErrorPatternIgnored(
+      pattern = "ModifyCacheMaxSizeAction:build()",
+      reason = "Node config cache is expected to be missing during shutdown")
+  @AErrorPatternIgnored(
+      pattern = "NodeConfigCollector:collectAndPublishMetric()",
+      reason = "Shard request cache metrics is expected to be missing")
+  @AErrorPatternIgnored(
+      pattern = "CacheUtil:getCacheMaxSize()",
+      reason = "Shard request cache metrics is expected to be missing.")
+  @AErrorPatternIgnored(
+      pattern = "HighHeapUsageYoungGenRca:operate()",
+      reason = "YoungGen metrics is expected to be missing.")
+  @AErrorPatternIgnored(
+      pattern = "PersistableSlidingWindow:<init>()",
+      reason = "Persistence base path can be null for integration test.")
```

Comment:
This is fixed with Sid's PR. We dont need to add them here

Reply:
thanks for catching this. Yes. let me remove those in my next PRs(I am creating a few more ITs to cover corner cases in JVM decider)

Reply:
done, removed this pattern